app/middleware/cors.py

```diff
@@ -33,6 +33,6 @@ def __call__(self, request):
         if request.path in (f'/{STAC_BASE}/v0.9/search', f'/{STAC_BASE}/v1/search'):
             allow_methods.append('POST')
         response['Access-Control-Allow-Methods'] = ','.join(allow_methods)
-        response['Access-Control-Allow-Headers'] = 'Content-Type,Accept'
+        response['Access-Control-Allow-Headers'] = 'Content-Type,Access,Accept-Language'
```

Comment:
:+1: for adding `Accept-Language`, but I don't understand why you suggest to remove `Accept` and add `Access`? I don't see `Access` header being sent by Edge (and to my knowledge it's not a standard/common header)